swarm-production/README.md

```diff
@@ -1,213 +1,220 @@
+Creating service zenko-prod_quorum
+Creating service zenko-prod_mongodb
+Creating service zenko-prod_queue
+Creating service zenko-prod_s3-front
+Creating service zenko-prod_lb
+Creating service zenko-prod_backbeat-consumer
+Creating service zenko-prod_backbeat-api
+Creating service zenko-prod_s3-data
+Creating service zenko-prod_backbeat-producer
+Creating service zenko-prod_cache
+Creating service zenko-prod_mongodb-init
+```
+
+Check that the services are up:
+
+```shell
+$ docker stack services zenko-prod
+ID                  NAME                           MODE                REPLICAS            IMAGE                          PORTS
+1j8jb41llhtm        zenko-prod_s3-data             replicated          1/1                 zenko/cloudserver:pensieve-3   *:30010->9991/tcp
+3y7vayna97bt        zenko-prod_s3-front            replicated          1/1                 zenko/cloudserver:pensieve-3   *:30009->8000/tcp
+957xksl0cbge        zenko-prod_mongodb-init        replicated          0/1                 mongo:3.6.3-jessie
+cn0v7cf2jxkb        zenko-prod_queue               replicated          1/1                 wurstmeister/kafka:1.0.0       *:30008->9092/tcp
+jjx9oabeugx1        zenko-prod_mongodb             replicated          1/1                 mongo:3.6.3-jessie             *:30007->27017/tcp
+o530bkuognu5        zenko-prod_lb                  global              1/1                 zenko/loadbalancer:latest      *:80->80/tcp
+r69lgbue0o3o        zenko-prod_backbeat-api        replicated          1/1                 zenko/backbeat:pensieve-4
+ut0ssvmi10tx        zenko-prod_backbeat-consumer   replicated          1/1                 zenko/backbeat:pensieve-4
+vj2fr90qviho        zenko-prod_cache               replicated          1/1                 redis:alpine                   *:30011->6379/tcp
+vqmkxu7yo859        zenko-prod_quorum              replicated          1/1                 zookeeper:3.4.11               *:30006->2181/tcp
+y7tt98x7jdl9        zenko-prod_backbeat-producer   replicated          1/1                 zenko/backbeat:pensieve-4
+[...]
+```
+
+**Note:** Having 0 replicas of the mongodb-init service is fine, because it is
+expected to execute successfully only once to initialize the mongodb replica
+set.
+
+## Using Zenko Orbit
+
+To get your instance's Zenko Orbit identifier and claim it in the portal,
+issue this command:
+
+```shell
+$ docker service logs zenko-prod_s3-front | grep -i instance \
+  zenko-prod_s3-front.1.khz73ag06k2k@moby | {"name":"S3","time":1512424260154,\
+  "req_id":"115779d9564e960048a5","level":"info","message":"this deployment's \
+   Instance ID is ce1bcdb7-8e30-4e3f-b7a2-9424078c9159","hostname": \
+   "843d31bf15f0", "pid":28}
+```
+
+Go to [Zenko Orbit](https://www.zenko.io/admin) to manage your deployment
+through a nifty UI.
+
+## Testing
+
+To use the `tests` folder, update the credentials in
+`Zenko/tests/utils/s3SDK.js` with credentials generated in Zenko Orbit.

```

Comment:
should be `Zenko/tests/backbeat/s3SDK.js`

Reply:
Also theres a boatload of evironment variables that need to be set for it to run